src/utils/eventDispatcher.test.ts

```diff
@@ -0,0 +1,62 @@
+import EventDispatcher from './eventDispatcher';
+
+describe('EventDispatcher', () => {
+  test('should emit event', () => {
```

Comment:
It might be good to replace `test` with `it`.
```suggestion
  it('should emit event', () => {
```

Reply:
It looks much better😀